client/components/pages/AuthPage/AuthPage.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react'
+import { Link, Redirect } from "react-router-dom";
+
+export const LOGIN = 'login';
+export const REGISTER = 'register';
+export const FORGOT = 'forgot';
+export const RESET = 'reset';
+
+const ResetPage = ({ isActive }) => isActive && (
+    <>
+        <form>
+            <p className="title" >Reset Password</p>
+            <input type="Password" placeholder="Password" className="inputField" />
```

Comment:
Add required prop to all input fields so that you can't submit without filling it

Reply:
This is comment is meant for the entire file not just this line